nengo_spinnaker/operators/lif.py

```diff
@@ -253,18 +315,35 @@ def write_subregion_to_file(self, fp, vertex_slice):
         )
         fp.write(data)
 
+PESLearningRule = collections.namedtuple(
+    "PESLearningRule", "learning_rate, filter_index, decoder_offset")
+
 
 class PESRegion(regions.Region):
     """Region representing parameters for PES learning rules.
     """
-    # TODO Implement PES
+    def __init__(self):
+        self.learning_rules = []
 
     def sizeof(self, *args):
-        return 4
+        return 4 + (len(self.learning_rules) * 12)
 
     def write_subregion_to_file(self, fp, vertex_slice):
-        # Write out a zero, indicating no PES data
-        fp.write(b"\x00" * 4)
+        # Get length of slice for scaling learning rate
+        n_neurons = float(vertex_slice.stop - vertex_slice.start)
+
+        # Write number of learning rules
+        fp.write(struct.pack("<I", len(self.learning_rules)))
+
+        # Write learning rules
+        for l in self.learning_rules:
+            data = struct.pack(
+                "<iII",
+                tp.value_to_fix(l.learning_rate / n_neurons),
+                l.filter_index,
+                l.decoder_offset
+            )
+            fp.write(data)
```

Comment:
I would say that: 
a) multiple learning rules are the exception rather than the rule and 
b) it'd be nicer to enable the rig system than start adding adhoc buffering